### ISSUE TITLE
Compiling error!

### ISSUE THREAD

**Issue by calex80** [adamsvoboda/nim-loader#5]
I have this error, can anyone help to figure it out?

```Error: execution of an external compiler program 'gcc -c  -w -fmax-errors=3 -Os -fno-ident   -I/usr/lib/nim/lib -I/home/kali/nim-loader -o /root/.cache/nim/loader_mod_r/@mevasion@sunhookNTDLL.nim.c.o /root/.cache/nim/loader_r/@mevasion@sunhookNTDLL.nim.c' failed with exit code: 1```

Thanks in advance

**Comment by rodry7** [comment]
Were you able to fix it?

**Comment by calex12** [comment]
Yes, I fixed it. I ran `nimble install winim nimcrypto docopt ptr_math strenc`
